pyramid_openapi3/exceptions.py

```diff
@@ -1,13 +1,13 @@
 """Exceptions used in pyramid_openapi3."""
 
-from openapi_core.schema.exceptions import OpenAPIError
+from dataclasses import dataclass
+from openapi_core.exceptions import OpenAPIError
 from openapi_core.unmarshalling.schemas.exceptions import InvalidSchemaFormatValue
 from pyramid.httpexceptions import HTTPBadRequest
 from pyramid.httpexceptions import HTTPInternalServerError
 from pyramid.request import Request
 from pyramid.response import Response
 
-import attr
```

Comment:
Can we remove attr from dependency list?

Reply:
Huh, we don't even have `attrs` in pyproject.toml. Nothing to do here, move along.